### DISCUSSION TITLE
Would it be possible to add an indication in PRs which one has comments?

### DISCUSSION THREAD

**Discussion by fombico** [community/community#46303] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When I am reviewing PRs, I add my pending comments and check the `Viewed` box, which collapses the file. However, before I submit my review, I sometimes want to go back and revisit my comments again. With large PRs and lots of files, I lose track of which files have comments in them. It would be nice to have some indication that a file has a comment, so it is easy to revisit them.

See below for an example:
![Screenshot 2023-02-03 at 12 13 06 PM](https://user-images.githubusercontent.com/1717428/216665509-e9a248b8-6c32-4b41-a54b-a6766ce45acd.png)

**Comment by soroshsabz** [comment]
I think it is very good feature and useful

**Comment by p7g** [comment]
By the end of a long review I often forget if any of my comments were blocking, so it would be great to be able to quickly see what they were to decide if I want to approve or not.

This suggestion would already be a great improvement, but even better would be a summary view of all the comments before submitting the review.
